source/2019-04-05-the-ember-times-issue-93.md

```diff
@@ -47,10 +47,17 @@ Side note 2. [If you haven't joined the Ember Community on Discord, do so today]
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [How Can We Improve?💡 Looking Ahead...👀🚀](https://emberjs.com/ember-community-survey-2019/#how-can-we-improve-ember)
+
+Of course the community survey highlights and reinforces so many of the aspects we all know and love about Ember.js, the community, the productivity, the stability! ♥️🎉✨ But, the survey also gives us as community members a chance to reflect on things we can do better. 🤔
+
+Specifically the survey empowers community members, gives us a voice and a chance to comment about [how we feel Ember can improve](https://emberjs.com/ember-community-survey-2019/#how-can-we-improve-ember). Some shout-outs this year include continuing to broaden and diversify opinions, lighter builds and less Ember-specific syntax.
+
+Also, some of the [excellent talks](https://www.youtube.com/playlist?list=PLE7tQUdRKcyYWLWrHgmWsvzsQBSWCLHYL) we heard at EmberConf in 2019, like [@melsumner](https://github.com/melsumner)'s amazing talk [Don't Break The Web](https://www.youtube.com/watch?v=41TDfT6bAs4&list=PLE7tQUdRKcyYWLWrHgmWsvzsQBSWCLHYL&index=15&t=0s), highlighted the importance of improving accessibility not just in Ember, but across the web as a whole.  
```

Comment:
Would you please update the link to my talk to https://noti.st/melsumner/Phhimm/dont-break-the-web (which includes both slides and the youtube video)? Thanks!

Reply:
Of course. Updating now!